Algorithm/Greedy/fractional_knapsack.cpp

```diff
@@ -0,0 +1,101 @@
+#include<bits/stdc++.h>
+using namespace std;
+void knapsack(int n,float weight[],float profit[],float items[],int knapsack_cap)
+{
+	float p=0.0;
+	bool flag[n];
+	float quantity[n];
+	int ub=0;//current weight in knapsack
+	//An array to store the quantity of ith item that are picked in descending order
+	for(int i=0;i<n;i++)
+		quantity[i]=0.0;
+	for(int i=0;i<n;i++)
+	{
+		if(ub+weight[i]<=knapsack_cap)
+		{
+			//If the value is less than the knapsack capacity,total item is considered
+			quantity[i]=1;
```

Comment:
Follow 4 space indentation throughout.

Reply:
Could you please brief me out on what exactly I have to do for 4 space indentation , or can I just dm you?